frontend/src/renderer/features/fileSplice.ts

```diff
@@ -0,0 +1,31 @@
+
+
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { type } from 'os';
+import { FileFormat, FileSpliceState, RootState } from 'renderer/Data';
+
+const initialState: FileSpliceState = {
+    files: [],
+  };
+
```

Comment:
can we move this to a file where all the interfaces are? like the `index.ts` you introduced? 

Would be cool if you'd make it consistent anyway